implementations/grafbase/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "fullName": "Grafbase",
+  "language": "Rust/Typescrpt",
+  "documentation": "https://grafbase.com/docs",
+  "type": "SDL first or Code first",
```

lets keep this consistent with other implementations
```suggestion
  "type": "Code first | SDL first",
```